pkg/integration/utils.go

```diff
@@ -27,6 +27,16 @@ func GetExitCode(err error) (int, error) {
 			return procExit.ExitStatus(), nil
 		}
 	}
+	// error if contained command not found/does not exist
```

Comment:
@duglin as I (now) understand it the pkg/integration is to set up/test the utilities used in integration-cli, so there's no need to add tests or change anything there.  The new exit codes are tested in int-cli.  Thnx for your patience :)